projects/lights_control/src/lights_periph.c

```diff
@@ -0,0 +1,60 @@
+#include "event_queue.h"
+#include "gpio.h"
+#include "lights_events.h"
+#include "structs.h"
+
+#define NUM_PERIPH_FRONT 4
+#define NUM_PERIPH_REAR 4
+
+static GPIOAddress s_front_address_lookup[] = {
```

Comment:
Maybe consolidate all your board-specific configuration into a `struct` instead of having them all scattered in here? @ckitagawa has them essentially all wrapped in a mono config `struct` in `chaos_config.c` if you want to look at his branch.